package.json

```
@@ -3,7 +3,7 @@
     "@emotion/react": "11.10.4",
     "@wireapp/antiscroll-2": "1.3.1",
     "@wireapp/avs": "8.2.16",
-    "@wireapp/core": "32.1.1",
+    "@wireapp/core": "^33.0.0",
```

Should be an exact match, no `^`
```suggestion
    "@wireapp/core": "33.0.0",
```

---

oops 👍